### ISSUE TITLE
Cannot Type with Pinyin Keyboard on the Fields

### ISSUE THREAD

**Issue by justinaquino** [luoliyan/chinese-support-redux#107]
**Describe the bug**
Cannot Type with Pinyin Keyboard on the Fields. 

**To Reproduce**
1. Installed the Add-on
2. Reset.
3. pressed A (to add a card)
4. Selected Chinese (Basic)
5. on New Card Dialogue, cannot input pinyin. 

**Expected behavior**
Can input　pinyin

**Screenshots**
![Screenshot 2019-12-09 15:42:48](https://user-images.githubusercontent.com/14933887/70416784-cc1d9d00-1a9a-11ea-997e-62d4da94b56d.png)

**Specs (please complete the following information):**
 - OS: Ubuntu 18.04
 - Anki Version 2.1.15
- Chinese Support Version v0.11.5

**Additional context**
Trying it out. Trying to Migrate from PLECO to Anki

**Comment by luoliyan** [comment]
I think this might be an input method issue. I'll have a play around on Fedora (which uses Gnome as well) and see if I can replicate. But there might not be much I can do.

**Comment by Golten** [comment]
I have the same problem. It began at about 6:00PM (Beijing Standard Time), December 17 2019. 

As I was populating flashcards by inputting chinese characters, autofill suddenly stopped. I tried the following solutions:

- restart anki
- delete and reinstall chinese redux
- delete and reinstall anki
- Restart computer

<img width="1439" alt="Screen Shot 2019-12-17 at 7 30 10 PM" src="https://user-images.githubusercontent.com/58978383/70992039-c4e23900-2103-11ea-8911-f40e1f4402c2.png">

Specs:

OS: Mac Mojave 10.14
Anki Version 2.1.15
Version: Chinese Redux 0130 (unsure how to check the version, 0.13.0?)

**Comment by luoliyan** [comment]
@Golten Yeah, it's 0.13.0. Anki strips out the dots in the version number.

Anyway, I think you have a different issue to the OP. In your screenshot, the add-on is not enabled for that note type. What happens if you click `汉字`?

**Comment by Golten** [comment]
Thank you! Grateful for your work supporting this app.
